### DISCUSSION TITLE
confluence -like features?

### DISCUSSION THREAD

**Discussion by walthar** [requarks/wiki#3152] [Help / Questions] [answered]
Confluence has some really nifty and useful editing features, which do increase productivity. I got accustomed to those with my previous employer and would like to see them in wiki.js too.
So, is there a way to auto-generate list of sub-pages with links to them? Another missed feature is ability to include page[s] content onto another page. If these are not supported now are those included in a roadmap?

**Comment by NGPixel** [comment, marked as answer]
This is coming in 3.0. See https://docs.requarks.io/releases
